diff -bur satyr-ubiquity-f50c546669f3/manage.py satyr-ubiquity-patched/manage.py
--- satyr-ubiquity-f50c546669f3/manage.py	2016-09-10 06:28:21 +0400
+++ satyr-ubiquity-patched/manage.py	2017-07-18 06:42:17 +0400
@@ -98,13 +98,13 @@
     options.profile_dir = find_profile_dir(options.profile)
     options.path_to_ext_root = os.path.join(options.my_dir, ext_subdir)
 
-    options.ext_id = get_install_rdf_property(options.path_to_ext_root, "id")
+    options.ext_id = get_install_rdf_property(options.path_to_ext_root, "em:id")
 
     options.ext_version = get_install_rdf_property(options.path_to_ext_root,
-                                                   "version")
+                                                   "em:version")
 
     options.ext_name = get_install_rdf_property(options.path_to_ext_root,
-                                                "name")
+                                                "em:name")
 
     if options.profile_dir:
         options.extension_file = os.path.join(options.profile_dir,
diff -bur satyr-ubiquity-f50c546669f3/ubiquity/install.rdf satyr-ubiquity-patched/ubiquity/install.rdf
--- satyr-ubiquity-f50c546669f3/ubiquity/install.rdf	2016-09-10 06:28:21 +0400
+++ satyr-ubiquity-patched/ubiquity/install.rdf	2017-07-18 06:52:46 +0400
@@ -1,46 +1,46 @@
 <?xml version="1.0"?>
-<rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
-  xmlns="http://www.mozilla.org/2004/em-rdf#">
-  <rdf:Description about="urn:mozilla:install-manifest">
-    <id>ubiquity@labs.mozilla.com</id>
-    <version>0.6.5b</version>
-    <type>2</type>
-    <unpack>true</unpack>
+<RDF xmlns="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+  xmlns:em="http://www.mozilla.org/2004/em-rdf#">
+  <Description about="urn:mozilla:install-manifest">
+    <em:id>ubiquity@labs.mozilla.com</em:id>
+    <em:version>0.6.5g</em:version>
+    <em:type>2</em:type>
+    <em:unpack>true</em:unpack>
 
     <!-- Target Application this extension can install into,
          with minimum and maximum supported versions. -->
-    <targetApplication><rdf:Description>
-      <!--Firefox--><id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</id>
-      <minVersion>47.0</minVersion>
-      <maxVersion>51.0a1</maxVersion>
-    </rdf:Description></targetApplication>
+    <em:targetApplication><Description>
+      <!--Firefox--><em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
+      <em:minVersion>47.0</em:minVersion>
+      <em:maxVersion>*</em:maxVersion>
+    </Description></em:targetApplication>
 
     <!-- Front End MetaData -->
-    <name>Ubiquity</name>
-    <description>An extension that allows for the use of dynamic commands in Firefox.</description>
-    <creator>Mozilla Corporation</creator>
-    <iconURL>chrome://ubiquity/skin/icons/ubiquibot-square.png</iconURL>
-    <homepageURL>https://wiki.mozilla.org/Labs/Ubiquity</homepageURL>
-    <optionsURL>chrome://ubiquity/content/preferences.xul</optionsURL>
+    <em:name>Ubiquity</em:name>
+    <em:description>An extension that allows for the use of dynamic commands in Firefox.</em:description>
+    <em:creator>Mozilla Corporation</em:creator>
+    <em:iconURL>chrome://ubiquity/skin/icons/ubiquibot-square.png</em:iconURL>
+    <em:homepageURL>https://wiki.mozilla.org/Labs/Ubiquity</em:homepageURL>
+    <em:optionsURL>chrome://ubiquity/content/preferences.xul</em:optionsURL>
 
-    <translator>egamonal, toniher (ca)</translator>
-    <translator>cers (da-DK)</translator>
-    <translator>cmos (de)</translator>
-    <translator>queltosh (es-ES)</translator>
-    <translator>JTojnar (cs-CZ)</translator>
-    <translator>AtteL (fi-FI)</translator>
-    <translator>ccorazza, Frederic, Goofy, myahoo (fr)</translator>
-    <translator>moZes (fy-NL)</translator>
-    <translator>kkemenczy (hu)</translator>
-    <translator>gialloporpora (it-IT)</translator>
-    <translator>mar, powchin, satyr (ja)</translator>
-    <translator>nathansamson (nl)</translator>
-    <translator>teo (pl-PL)</translator>
-    <translator>camponez (pt)</translator>
-    <translator>brunomagrani, fernph (pt-BR)</translator>
-    <translator>mrfyda (pt-PT)</translator>
-    <translator>Quicksilver tears, SeLarin (ru-RU)</translator>
-    <translator>yfdyh000 (zh-CN)</translator>
-    <translator>kourge (zh-TW)</translator>
-  </rdf:Description>
-</rdf:RDF>
+    <em:translator>egamonal, toniher (ca)</em:translator>
+    <em:translator>cers (da-DK)</em:translator>
+    <em:translator>cmos (de)</em:translator>
+    <em:translator>queltosh (es-ES)</em:translator>
+    <em:translator>JTojnar (cs-CZ)</em:translator>
+    <em:translator>AtteL (fi-FI)</em:translator>
+    <em:translator>ccorazza, Frederic, Goofy, myahoo (fr)</em:translator>
+    <em:translator>moZes (fy-NL)</em:translator>
+    <em:translator>kkemenczy (hu)</em:translator>
+    <em:translator>gialloporpora (it-IT)</em:translator>
+    <em:translator>mar, powchin, satyr (ja)</em:translator>
+    <em:translator>nathansamson (nl)</em:translator>
+    <em:translator>teo (pl-PL)</em:translator>
+    <em:translator>camponez (pt)</em:translator>
+    <em:translator>brunomagrani, fernph (pt-BR)</em:translator>
+    <em:translator>mrfyda (pt-PT)</em:translator>
+    <em:translator>Quicksilver tears, SeLarin (ru-RU)</em:translator>
+    <em:translator>yfdyh000 (zh-CN)</em:translator>
+    <em:translator>kourge (zh-TW)</em:translator>
+  </Description>
+</RDF>
diff -bur satyr-ubiquity-f50c546669f3/ubiquity/modules/parser.js satyr-ubiquity-patched/ubiquity/modules/parser.js
--- satyr-ubiquity-f50c546669f3/ubiquity/modules/parser.js	2016-09-10 06:28:21 +0400
+++ satyr-ubiquity-patched/ubiquity/modules/parser.js	2017-07-18 00:37:47 +0400
@@ -316,7 +316,7 @@
   get displayHtml() {
     var {escapeHtml} = Utils;
     var {matchedName, args} = this._verb;
-    var html = '<span class="verb">' + escapeHtml(matchedName) + "</span> ";
+    var html = '<span class="verb">' + escapeHtml(matchedName).replace(/-/g, " ") + "</span> ";
     for (let x in (this.fromNounFirstSuggestion
                    ? this._argSuggs
                    : this._argStrings)) {
diff -bur satyr-ubiquity-f50c546669f3/ubiquity/modules/setup.js satyr-ubiquity-patched/ubiquity/modules/setup.js
--- satyr-ubiquity-f50c546669f3/ubiquity/modules/setup.js	2016-09-10 06:28:21 +0400
+++ satyr-ubiquity-patched/ubiquity/modules/setup.js	2017-07-18 00:35:56 +0400
@@ -85,6 +85,9 @@
   }, {
     source: "search.xhtml",
     title: "Mozilla Web Search Commands",
+  }, {
+    source: "local.js",
+    title: "Subscribe Commands Locally",
   }],
 
   __maybeReset: function __maybeReset() {
Only in satyr-ubiquity-patched/ubiquity/standard-feeds: local.js
Only in satyr-ubiquity-patched: ubiquity-0.6.5g.xpi
